plugins/extractors/bigquery/bigquery.go

```diff
@@ -167,7 +195,7 @@ func (e *Extractor) buildTable(ctx context.Context, t *bigquery.Table, md *bigqu
 
 	return &assets.Table{
 		Resource: &common.Resource{
-			Urn:     models.TableURN("bigquery", t.ProjectID, t.DatasetID, t.TableID),
+			Urn:     tableURN,
```

Comment:
while we're at it, can you help populating Description here as well 🙏 
```suggestion
			Urn:     tableURN,
			Description: md.Description,
```